lib/identifiers/isbn.rb

```diff
@@ -9,21 +9,32 @@ def self.extract(str)
     end
 
     def self.extract_isbn_as(str)
-      extract_thirteen_digit_isbns(str.scan(REGEX_A).join("\n").tr('/.', ''))
+      isbn = str.to_s.scan(REGEX_A).join("\n").tr('/.', '')
```

Comment:
Minor naming thing but the result of this line is not necessarily an ISBN: it is a string of candidate ISBN-As which we then extract ISBN-13s from.

Perhaps `isbn_as` is a better name?